web/src/page_params.ts

```diff
@@ -9,6 +11,7 @@ export const page_params: {
         name: string;
         allowed: boolean;
     }[];
+    contributors: Contributor[] | undefined;
```

Comment:
This remains wrong. The point is that `page_params` has a different type depending on what page it is. The tiny bit of code in this module that isn't declaring a type is literally one line of code. We should just put copies of that one line of code in each module that's actually using it exporting the correct types for that module's `page_params`. 

No page ever has the type you're changing this to -- the teams page only has `contributors` and maybe a couple other elements; the app doesn't have a `contributors`, and the teams page doesn't have a `max_avatar_file_size_mib`. 

(Or, better, we should just use `zod` to parse the `page_params` and do that at the top of individual modules like `teams_page.ts`, so we're actually validating the types).